aiosonic/__init__.py

```diff
@@ -222,11 +222,7 @@ async def read_chunks(self) -> AsyncIterator[bytes]:
 
     def __del__(self):
         # clean it
-        if self.chunked and not self.chunks_readed:
-            loop = None
-            if self.connection:
-                loop = get_loop()
-                loop.create_task(self.connection.release())
+        pass
```

Comment:
I think you can just remove the whole method

Reply:
Will do so. Thank you for your kindness.